src/opera/commands/package.py

```diff
@@ -0,0 +1,68 @@
+import argparse
+import uuid
+
+from pathlib import Path
+
+from opera.error import DataError, ParseError
+from opera.parser.tosca.csar import CloudServiceArchive
+
+
+def add_parser(subparsers):
+    parser = subparsers.add_parser(
+        "package",
+        help="Package service template and all accompanying files into a CSAR"
+    )
+    parser.add_argument(
+        "--service-template", "-t",
+        help="Name or path to the TOSCA service template "
+             "file from the root of the input folder",
+    )
+    parser.add_argument(
+        "--output", "-o",
+        help="Output CSAR file path",
+    )
+    parser.add_argument(
+        "--format", "-f", choices=("zip", "tar"),
+        default="zip", help="CSAR compressed file format",
+    )
```

Comment:
Are tars conformant to the tosca standard?

Reply:
Well, I don't know actually.

TOSCA standard says [here](https://docs.oasis-open.org/tosca/TOSCA/v1.0/os/TOSCA-v1.0-os.html#_Toc356403711): `A CSAR is a zip file containing at least two directories, the TOSCA-Metadata directory and the Definitions directory. `, but it doesn't prevent other compression formats explicitly. I think I saw some tar compressed CSARs in RADON but I'm not sure. And if I search a little bit more: [here](https://blueprints.launchpad.net/heat-translator/+spec/tosca-csar-translation), it is said that `TOSCA template files most likely will be provided as part of CSAR archive file (typically zip or tar).`, or if you look at [toscafy CSAR tool](https://github.com/toscafy/toscafy/blob/6fa7e24f75503cdc536aec3b97475f342fb21377/swagger.yaml#L69) it offers packing the CSAR files with zip or tar compression formats. It seems that [Aria TOSCA orchestrator](http://ariatosca.incubator.apache.org/docs/html/aria.cli.html#aria.cli.service_template_utils.get) also uses different compression formats. @matejart do you happen to know this?

Reply:
We decided to keep both zip or tar CSAR formats because TOSCA doesn't specify any exact mimetype here.